policy/release/buildah_build_task.rego

```diff
@@ -33,7 +33,7 @@ deny contains result if {
 	buildah_tasks
 	some buildah_task in buildah_tasks
 	not lib.tkn.task_param(buildah_task, "DOCKERFILE")
-	name := lib.tkn.task_name(buildah_task)
+	name := lib.tkn.pipeline_task_name(buildah_task)
 	result := lib.result_helper_with_term(rego.metadata.chain(), [name], name)
```

Comment:
I just want to clarify that `pipeline_task_name` should only ever be used to provide additional context to the user as in an error message, for example. It should never be used when asserting that a particular Task was provided.

This is important because the Pipeline Task name is a property of the Pipeline definition, while a Task name is a property of the Task definition.

I agree that in this line, the error message should likely include the Pipeline Task name. But the `term` (third parameter) should continue to be the Task name. Besides the reason I mentioned above, switching the term would likely also require a change to the `include`/`exclude` options in existing policy configs.

Here, I would consider an error message that specifies both the Pipeline Task and the Task names while keeping the `term` as the Task name.

Reply:
Good point. I updated the error message to include the pipeline task and the underlying task name.